src/overlays/actors/ovl_En_Rz/z_en_rz.c

```diff
@@ -559,15 +559,12 @@ void func_80BFC674(EnRz* this, PlayState* play) {
 
     if (Actor_ProcessTalkRequest(&this->actor, &play->state)) {
         this->actionFunc = func_80BFC608;
-        if (Player_GetMask(play) == PLAYER_MASK_KAMARO) {
+        if (Player_GetMask(play) == 0xE) {
```

Comment:
```suggestion
        if (Player_GetMask(play) == PLAYER_MASK_KAMARO) {
```
I only omitted it to avoid having to dig out the enum.